cli/cluster/membership.go

```diff
@@ -20,28 +20,40 @@ type MembershipInstance struct {
 
 type MembershipInstances map[string]*MembershipInstance
 
+type InstancesFilter func(*MembershipInstances, map[string]string)
+
 func (membershipInstance *MembershipInstance) DecodeMsgpack(d *msgpack.Decoder) error {
 	return common.DecodeMsgpackStruct(d, membershipInstance)
 }
 
+// GetFirstInstanceName returns first instance name from the map.
+// Empty string is returned if the map is empty.
+func GetFirstInstanceName(instances map[string]string) string {
+	for _, instanceName := range instances {
```

Comment:
I'm surprised, but this is really typical code for getting a random item from a map)
I don't know how much the wording "first" is fair for the map, but that's up to you.

Reply:
I was thinking about the same: "random" instead of "first". Done.